modules/rosa-policy-security-regulation-compliance.adoc

```diff
@@ -57,15 +57,17 @@ Any issues that may be discovered are prioritized based on severity. Any issues
 
 | HIPAA Qualified | Yes | No
 
-| ISO 27001 | Yes | No
+| ISO 27001 | Yes | Yes
 
-| ISO 27017 | Yes | No
+| ISO 27017 | Yes | Yes
 
-| ISO 27018 | Yes | No
+| ISO 27018 | Yes | Yes
 
-| PCI DSS | Yes | No
+| PCI DSS | Yes | Yes
 
-| SOC 2 Type 2 | Yes | No
+| SOC 1 Type 2 | Yes | No
+
+| SOC 2 Type 2 | Yes | Yes
 
 | SOC 3 | Yes | No
```

Comment:
I confirmed that SOC3 should also be yes for ROSA with HCP

Reply:
Updated, thank you!